src/Emogrifier.php

```diff
@@ -339,47 +339,47 @@ public function emogrify()
      */
     public function emogrifyBodyContent()
     {
-        $this->xmlDocument = $this->createXmlDocument();
+        $this->domDocument = $this->createDomDocument();
 
-        $processedXmlDocument = $this->process($this->xmlDocument);
-        $bodyNodeHtml = $processedXmlDocument->saveHTML($this->getBodyElement($processedXmlDocument));
+        $processedDomDocument = $this->process($this->domDocument);
+        $bodyNodeHtml = $processedDomDocument->saveHTML($this->getBodyElement($processedDomDocument));
 
         return \str_replace(['<body>', '</body>'], '', $bodyNodeHtml);
     }
 
     /**
-     * Creates an XML document from $this->html.
+     * Creates an DOM document from $this->html.
```

Comment:
How is it correct: "a**n** DOM ..." or "a DOM document"?

Reply:
Oh, this needs to be "a DOM …". Good catch! I'll repush in a minute.

Reply:
Fixed and repushed.